src/PresentationalComponents/AdvisoryType/AdvisoryType.js

```diff
@@ -1,16 +1,17 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { advisoryTypes } from '../../Utilities/constants';
+import { Flex, FlexItem } from '@patternfly/react-core';
 
 const AdvisoryType = ({ type }) => {
     const advisoryType =
         advisoryTypes.find(item => item.value === type) || advisoryTypes[3];
     return (
         <React.Fragment>
-            <div className={'icon-with-label'}>
```

Comment:
Aren't we ready to remove the class "icon-with-label" completely when using Flex component? It may be at other places too.